### PR TITLE
COMMERCE-11923 added POA column to PL table

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/COMMERCE-11923

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/data/set/view/table/CPInstancePriceEntryTableFDSView.java [hunk 1]
```diff
@@ -39,6 +39,10 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 			"unitPrice", "unit-price"
 		).add(
 			"createDate", "create-date"
+		).add(
+			"priceOnApplication", "price-on-application",
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/data/set/view/table/CPInstancePriceEntryTableFDSView.java:43]: is there a reason for this sorting? This is the order in which the columns will be shown in the UI, so isn't better to have the createDate at the end (evantually with workflow fields) like other similar tables?

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/data/set/view/table/CPInstancePriceEntryTableFDSView.java (hunk 1 above)
Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/data/set/view/table/CPInstancePriceEntryTableFDSView.java:41]: done

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
QA passed. Add coverage and resend on https://github.com/liferay-commerce/liferay-portal/pull/4175

**Comment by jaydawu** [comment]
ci:close

**Pull request closed without merging**